### PR TITLE
Adding user namespace in README

### PR DESCRIPTION
**Pull request by rajasec**
Updating README file with user namespace.

Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
Just adding user namespace won't work. You will also need to add the mappings. Maybe create a separate section for a working user namespaces example?

**Comment by jfrazelle** [comment]
I have a lot of examples here if you need ideas
https://github.com/jfrazelle/containers/ 0:)

**Comment by rajasec** [comment]
@mrunalp 
I have added the respective mappings for user name space. please review

File: libcontainer/README.md [hunk 1]
```diff
@@ -129,6 +130,20 @@ config := &configs.Config{
 			Flags:       defaultMountFlags | syscall.MS_RDONLY,
 		},
 	},
+	UidMappings: []configs.IDMap{
+		{
+			ContainerID: 0,
+			Host: 1000,
+			size: 10,
```
Comment (mrunalp) [inline libcontainer/README.md]: I would increase the size, for e.g. 65536

**Comment by rajasec** [comment]
@mrunalp 
Updated. the size.

**Comment by hqhq** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**